lib/assembly/application.ex

```diff
@@ -7,9 +7,24 @@ defmodule Assembly.Application do
 
   require Logger
 
+  import Cachex.Spec
+
   def start(_type, _args) do
     children = [
-      {Assembly.Broadway, []}
+      {Assembly.Broadway, []},
+      # For populating the Cache we have a few options:
+      # 1. We can handle this ourselves
+      # 2. Investigate and possibly leverage Cachex's warming functionality
+      #   https://hexdocs.pm/cachex/reactive-warming.html#content
+      #   https://hexdocs.pm/cachex/proactive-warming.html#content
+      {Cachex,
```

Comment:
Does ordering matter for this stuff? I've noticed some weird things on some of my other projects. Assuming it does, this should be `DynamicSupervisor` `Cache` and then `Broadway`.

Reply:
I reordered these following your suggestion :+1: